backend/src/main/java/com/sim_backend/websockets/observers/AuthorizeObserver.java

```diff
@@ -0,0 +1,42 @@
+package com.sim_backend.websockets.observers;
+
+import com.sim_backend.websockets.OCPPWebSocketClient;
+import com.sim_backend.websockets.enums.AuthorizationStatus;
+import com.sim_backend.websockets.events.*;
+import com.sim_backend.websockets.messages.*;
+
+public class AuthorizeObserver implements OnOCPPMessageListener {
+
+  public void handleAuthorize(OCPPWebSocketClient client, AuthorizeResponse response) {
```

Comment:
Per the OCPP spec, the Authorize operation is only initiated by the charge point. Therefore, our simulator will never send an `AuthorizeResponse` message. 

Instead, this function can just send an `Authorize` message.